_dockerPublisher/Dockerfile

```diff
@@ -11,35 +11,27 @@ MAINTAINER "Michael Biarnes Kiefer <mbiarnes@redhat.com>"
 ### swith to root user ###
 USER root
 
-### install ruby and rake ###
-RUN dnf install -y ruby
-RUN dnf install -y rubygems
-RUN dnf install -y rubygem-rake
-RUN dnf install -y rubygem-bundler
-RUN dnf install -y ruby-devel
-RUN dnf install -y rpm-build
-RUN dnf install -y gcc-c++
-RUN dnf install -y make
-RUN dnf install -y rsync
-RUN dnf install -y openssh-clients
-RUN dnf install -y net-tools
-RUN dnf install -y iputils
-RUN dnf install -y wget
+### Install ruby dependencies
+RUN dnf install -y ruby rubygems rubygem-rake rubygem-bundler ruby-devel
+
+## Install tools
+RUN dnf install -y rpm-build gcc-c++ make rsync openssh-clients net-tools iputils wget
+
```

Comment:
so why didn't we go into more optimization and do install of packes in one line only?

Reply:
Because I thought it is more orderly if we have a division by ruby  and other stuff. Off course we can do the installation in one line.

Reply:
Yes, i got your point. You still can have comments there and just visually split in two lines while there can be only one command line.

Reply:
@mareknovotny DONE

Reply:
DONE